lms/static/js/dialog_tab_controls.js

```diff
@@ -0,0 +1,77 @@
+var DialogTabControls = (function() {
+    'use strict';
+
+    var focusableChildren,
+        numElements,
+        currentIndex,
+        setCurrentIndex = function(currentElement) {
```

Comment:
@cahrens For these modals focus was going out of the modal, to fix that we handled it with javascript thats why we added this.